runtime_tests/build.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
I've just realized this script and the `runAll.js` one are in a brand new `runtime_tests` directory.
I'd rather these were kept in the already existing `test/com/google/javascript/jscomp/runtime_tests` directory.
Maybe put them in a `scripts` subdirectory there?

Reply:
I moved them to a directory called `utils` inside the `runtime_tests` dir. This actually gave me an opportunity to correct a business logic error I missed - only ~100 tests were being executed when there were ~300. Should be good now.

Reply:
I moved that folder from `./runtime_tests` to `./tests/.../runtime_tests/utils`.